packages/torii-wasm/node.mjs

```diff
@@ -0,0 +1,29 @@
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
+const dojo_c = require("./pkg/node/dojo_c.js");
+
+// Re-export all named exports
+export const {
```

Comment:
Wouldn't this add more overhead on updates? If we were to add new types and we'd need to every time update the named exports